src/components/pipelineRuns-list/PipelineRunsList.tsx

```diff
@@ -70,6 +76,7 @@ const PipelineRunsList: React.FC<PipelineRunsListProps> = ({
         rowData={{
           taskRuns,
           taskRunsLoaded,
+          repositoryPLRs,
```

Comment:
<img width="1438" alt="Screenshot 2024-04-09 at 6 06 46 PM" src="https://github.com/openshift-pipelines/console-plugin/assets/102503482/e25686d9-1f85-4726-bfdd-8451f82fb2f4">

For some reason top spacing is removed again

Reply:
Fixed it.